ChaosMod/Effects/db/Vehs/VehsBusJesus.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+	Effect by ProfessorBiddle, a very shite programmer
+*/
+
+#include <stdafx.h>
+
+static void OnStart()
+{
+	Ped playerPed = PLAYER_PED_ID();
+
+	//delete current vehicle
+	if (IS_PED_IN_ANY_VEHICLE(playerPed, false))
+	{
+		Vehicle playerVeh = GET_VEHICLE_PED_IS_IN(playerPed, false);
+		Vector3 vel = GET_ENTITY_VELOCITY(playerVeh);
+		Vector3 oldPos = GET_ENTITY_COORDS(playerVeh, false);
+		SET_ENTITY_COORDS_NO_OFFSET(playerVeh, oldPos.x, oldPos.y, oldPos.z + 5, 0, 0, 0);
+		SET_ENTITY_VELOCITY(playerVeh, vel.x, vel.y, 40.f);
+	}
+
+	static constexpr Hash modelHash = -835930287;
+	LoadModel(modelHash);
+
+	Hash relationshipGroup;
+	ADD_RELATIONSHIP_GROUP("_WHEEL_JESUS", &relationshipGroup);
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, relationshipGroup, GET_HASH_KEY("PLAYER"));
+
+	//spawn bus and set player into bus
+	Vector3 spawnPos = GET_ENTITY_COORDS(PLAYER_PED_ID(), false);
+	Vehicle veh = CreatePoolVehicle(GET_HASH_KEY("BUS"), spawnPos.x, spawnPos.y, spawnPos.z, GET_ENTITY_HEADING(PLAYER_PED_ID()));
+	SET_PED_INTO_VEHICLE(playerPed, veh, -2);
+
+	//put jesus behind wheel
+	Ped jesus = CREATE_PED_INSIDE_VEHICLE(veh, 4, modelHash, -1, true, false);
```

Comment:
Use `CreatePoolPed` instead. There isn't a handy function for creating them directly inside a vehicle but it's a simple `SET_PED_INTO_VEHICLE` call afterwards anyways.